services/db/workspaceTableHandler.go

```diff
@@ -0,0 +1,104 @@
+package db
+
+import (
+	"database/sql"
+	"fmt"
+	"log"
+	"strconv"
+
+	"github.com/rudderlabs/rudder-server/config"
+)
+
+var (
+	host, user, password, dbname string
+	port                         int
+)
+
+type WorkspaceDataT struct {
+	token      string
+	created_at string
+	parameters string
+}
+
+func loadOriginalDBConfig(originalDBname string) {
+	host = config.GetEnv("JOBS_DB_HOST", "localhost")
+	user = config.GetEnv("JOBS_DB_USER", "ubuntu")
+	dbname = originalDBname
+	port, _ = strconv.Atoi(config.GetEnv("JOBS_DB_PORT", "5432"))
+	password = config.GetEnv("JOBS_DB_PASSWORD", "ubuntu") // Reading secrets from
+
+}
+
+func loadDBConfig() {
+	host = config.GetEnv("JOBS_DB_HOST", "localhost")
+	user = config.GetEnv("JOBS_DB_USER", "ubuntu")
+	dbname = config.GetEnv("JOBS_DB_DB_NAME", "ubuntu")
+	port, _ = strconv.Atoi(config.GetEnv("JOBS_DB_PORT", "5432"))
+	password = config.GetEnv("JOBS_DB_PASSWORD", "ubuntu") // Reading secrets from
+
+}
+
+func createWorkspaceTable(dbHandle *sql.DB) {
```

Comment:
We already have this function in envValidator.go?

Reply:
i am getting import cycle errors.. i have tried to import those packages and use existing functions.
ill look into it